README.md

```diff
@@ -13,6 +13,7 @@ The global-workflow current supports the following machines:
 * WCOSS-Cray
 * Hera
 * Orion
+* S4
```

Comment:
I think we need to distinguish between tier-1 support and tier-2 support here (we also don't support the Cray anymore, but that's not your fault).

Reply:
Good point, I've specified S4 as a tier-2 machine, and deleted Cray.